index.php

```diff
@@ -778,19 +784,6 @@ function drawmap() {
                 //     resFactor: 1
                 // });
 
-                // TODO oli
-                // var poiLayerWikipediaHttp = new OpenLayers.Protocol.HTTP({
-                //     url: 'api/proxy-wikipedia.php?',
-                //     params: {
-                //         'LANG' : language,
-                //         'thumbs' : 'no'
-                //     },
-                //     format: new OpenLayers.Format.KML({
-                //         extractStyles: true,
-                //         extractAttributes: true
-                //     })
-                // });
-
                 // Select feature ---------------------------------------------------------------------------------------------------------
                 // (only one SelectFeature per map is allowed)
                 // TODO oli
```

Comment:
Is this still a to-do?

Reply:
No the Select control is not needed anymore

Reply:
basically all the TODO are done I think . I just waited the tests from Markus to remove theme